app/src/main/res/values/strings.xml

```diff
@@ -1366,4 +1368,20 @@
     <string name="voice_search_content_description">Voice search</string>
     <!-- Voice search prompt description displayed after the user presses the voice search button -->
     <string name="voice_search_explainer">Speak now</string>
+
+    <!-- Synced Tabs -->
+    <string name="synced_tabs_connect_to_sync_account">Connect with a Firefox Account.</string>
+    <string name="synced_tabs_connect_another_device">Connect another device.</string>
+    <string name="synced_tabs_reauth">Please re-authenticate.</string>
+    <string name="synced_tabs_enable_tab_syncing">Please enable tab syncing.</string>
+    <string name="synced_tabs_sign_in_button">Sign in to see synced tabs</string>
+    <!-- Sync Tabs overflow menu copy button -->
+    <string name="sync_tabs_menu_copy_button">Copy</string>
```

Comment:
Okay I think you made these strings for the menu, but didn't actually use their resources when setting the menu up.